src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    FIRST_PRIZE(6, false, 2_000_000_000),
+    SECOND_PRIZE(5, true, 30_000_000),
+    THIRD_PRIZE(5, false, 1_500_000),
+    FOURTH_PRIZE(4, false, 50_000),
+    FIFTH_PRIZE(3, false, 5_000),
+    MISS(0, false, 0);
+
+    private final int matchCounts;
+    private final boolean isBonusBall;
+    private final int prizeMoney;
+
+    private LottoRank(int matchCounts, boolean isBonusBall, int prizeMoney) {
+        this.matchCounts = matchCounts;
+        this.isBonusBall = isBonusBall;
+        this.prizeMoney = prizeMoney;
+    }
```

Comment:
enum은 생성자를 디폴트로 사용합니다! private일 필요는 없어요! 
```suggestion
    LottoRank(int matchCounts, boolean isBonusBall, int prizeMoney) {
        this.matchCounts = matchCounts;
        this.isBonusBall = isBonusBall;
        this.prizeMoney = prizeMoney;
    }
```